project-base/storefront/public/locales/cs/common.json

```diff
@@ -261,6 +266,7 @@
     "Preferences": "Preferenční",
     "Previous": "Předchozí",
     "Previous products": "Předchozí produkty",
+    "price": "cena",
     "Price": "Cena",
```

Comment:
Same here. Seems weird to have 2 exact same strings, only with a different capital letter.

Reply:
Fixed